### PR TITLE
added dropbox.py

### PR DESCRIPTION
**Pull request by tjaartvdwalt**
I added a module to check your dropbox status.

The command I use to get the status is dropbox-cli (on Arch Linux at least). I am not sure if this is portable.

### CODE REVIEW

**Comment by Horgix** [comment]
What do you mean by "status" ? Asking this for curiosity sake only :)
Is it if the dropbox servers are reachable, or space remaining ? Maybe give us an example output ?

I'm asking that because it's absolutely not guessable from your code. Maybe changing [L.18](https://github.com/tjaartvdwalt/py3status/commit/295d070044d8fecf579156d7e4f28eee72423532#diff-24347ad1c1898def03d36ae716a30635R18) from `format = "Dropbox: {}"` to `format = "Dropbox: {remainingSpace}"` or `format = "Dropbox: {reachable}"` and [L.26](https://github.com/tjaartvdwalt/py3status/commit/295d070044d8fecf579156d7e4f28eee72423532#diff-24347ad1c1898def03d36ae716a30635R26) accordingly would make it easier to understand what's being retrieved, and easier to customize by users ?

Again, these comments are only from my own point of view and I'm sure your module will be welcomed as it is !

File: py3status/modules/dropbox.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+Display your dropbox status.
+
+Configuration parameters:
+    - format : Prefix text for the dropbox status
+
+Requires:
+    - the 'dropbox-cli' command
+
+@author Tjaart van der Walt (github:tjaartvdwalt)
+@license GNU GPL  http://www.gnu.org/licenses/gpl.html
```
Comment (ultrabug) [inline py3status/modules/dropbox.py]: Would you please agree to contribute this module under the BSD license ?

**Comment by ultrabug** [comment]
Thanks a lot for your contribution @tjaartvdwalt !

Few points:
* I added a comment about the license
* Can you confirm that it works for both python2 and python3 ?
* Please make sure your code pass a `flake8` parsing (or a `yapf` even better)

I think @Horgix has a point, the name of the module `dropbox` is not quite meaningful, but looking at the code I guess this modules checks that dropbox is running or not right ?

Cheers

**Comment by tjaartvdwalt** [comment]
I added a comment about the license
BSD license will be fine with me, I will change that.

I use this plugin mostly to make sure that dropboxd didn't die on me. This used to bea big issue for me when my laptop switches wireless networks, but recently I have not had any problems (I now use systemd to manage dropbox).

Examples of the values returned by `dropbox-cli status` includes:
`Dropbox isn't running`
`Starting`
`Syncing "filename"`
`Up to date`

**Comment by tjaartvdwalt** [comment]
Replying to @ultrabug comments:

> I added a comment about the license

BSD is fine with me, I will change that

> Can you confirm that it works for both python2 and python3 ?

It works for python3. I have also tested it for python2 by running it from the command line, and it appears to work.

> Please make sure your code pass a flake8 parsing (or a yapf even better)

I develop using flake8 parsing, so I can confirm it passes that.

> I think @Horgix has a point, the name of the module dropbox is not quite meaningful, but looking at the code I guess this modules checks that dropbox is running or not right ?

I am more than willing to change the name. Would `dropbox_status` make more sense?

**Comment by tjaartvdwalt** [comment]
To give you the rationale behind creating the module: I mainly use this to see if `dropboxd` is still running. It happened a few times to me that the service died, and files I thought got synced did not.

The values that gets returned by the `dropbox-cli status` command, that gets printed by this module includes:
`Dropbox isn't running!`
`Starting...`
`Downloading  file list...`
`Syncing "filename"`
`Up to date` 
 
I am new to submitting pull requests. Should I make the changes and submit a new pull request, or is there a way to ammend my current request?

**Comment by ultrabug** [comment]
Thanks for your rationale and explanation mate !

> I am more than willing to change the name. Would dropbox_status make more sense?

Maybe `dropboxd_status` please ?

> I am new to submitting pull requests. Should I make the changes and submit a new pull request, or is there a way to ammend my current request?

Just commit and push to your current fork, this will update this pull request, no need to amend or make a new one.

Cheers

**Comment by ultrabug** [comment]
Sorry for the delay @tjaartvdwalt and thank you again for your contribution. I'll do some QA on the code and rename it to dropbox_status to make it more clear.

**Pull request merged**